databases/business-sync/fragment-shoebox-item.js

```diff
@@ -62,6 +62,74 @@
           }
         }
       }
+    },
+    annotations: {
+      type: 'array',
```

Comment:
Having this structured as a simple array makes searching through and updating annotations tricky when using the data structure.  After some trial use and more discussion, the proposal of using a map of annotation type to an array of annotations seemed more attractive because it allows annotations relevant to a given context to be looked up more easily.